### ISSUE TITLE
Displaying results based on multi value Managed Metadata field

### ISSUE THREAD

**Issue by dimi-be** [microsoft-search/pnp-modern-search#1243]
Hi,

We have a use case where we want to use the Search Results Web Part to display show related pages after the contents of that page.

We do this using a multi value managed metadata page property called "Tags". We've entered the following query in the properties of the search results webpart: "owstaxIdTags: {Page.Tags.Label} NOT Title:{Page.Title}".

The above setup works when the page containing the Search Results Web Part has only one tag. When that page has multiple tags it doesn't display any results any more, I assume this is because the above search query will only return the pages matching all of the selected tags.

Is there anything we can do the get to our expected result?

Regards,
Dimitri

**Comment by jhholm** [comment]
Please see documentation linked below for futher information. Quickly testing the SharePoint OTB search way seems to work in PnP Modern Search as well.
https://docs.microsoft.com/en-us/sharepoint/technical-reference/query-variables#query-variables-with-multiple-values

TLDR; {|owstaxIdTags:{Page.Tags.Label} NOT Title:{Page.Title}. You just need to add the pipe operator.

**Comment by jhholm** [comment]
And a note, if you check the call from your browsers network logs, you can see the following.
e.g. Page has 2 tags set. The serialized query should be ((owsTaxIdTags;\"Tag 1\") OR (owstaxIdTags:\"Tag \")). So in this case there is an implicit or.

**Comment by dimi-be** [comment]
Wow, this is great, just what we needed!

Thank you so much for this tip, this will be useful in the future.
